src/components/metrics/card/SensitivitySection/SensitivitySection.js

```diff
@@ -0,0 +1,113 @@
+}
+
+class Plot extends Component {
+  render() {
+    const {xSamples, ySamples, size, xLabel, yLabel} = this.props
+    const data = [{ customValues: _.zip(xSamples, ySamples) }];
+    const valuesAccessor = (s) => s.customValues
+    const xAccessor = (e) => e[0]
+    const yAccessor = (e) => e[1]
+
+    if (size === 'SMALL'){ return(
+          <ScatterPlot
+            data={data}
+            width={180}
+            height={68}
+            margin={{top: 5, bottom: 9, left: 5, right: 3}}
+            xAxis={{tickArguments: [0], innerTickSize: 1, outerTickSize: 1, tickPadding: 1}}
+            yAxis={{tickArguments: [0], innerTickSize: 0, outerTickSize: 0, tickPadding: 0}}
+            x={xAccessor}
+            y={yAccessor}
+            values={valuesAccessor}/>
+    )}
+    else { return(
+      <ScatterPlot
+        data={data}
+        width={500}
+        height={300}
+        margin={{top: 10, bottom: 40, left: 60, right: 20}}
+        xAxis={{tickArguments: [6], innerTickSize: 5, outerTickSize: 2, tickPadding: 3, label: xLabel}}
+        yAxis={{tickArguments: [6], innerTickSize: 5, outerTickSize: 2, tickPadding: 3, label: yLabel}}
+        x={xAccessor}
+        y={yAccessor}
+        values={valuesAccessor}/>
+    )}
+  }
+}
+
+class RegressionStats extends Component {
+  render() {
+    const {xSamples, ySamples, size} = this.props
+    const regression = !!xSamples.length ? everpolate.linearRegression(xSamples, ySamples) : false
+    const sampleSize = xSamples && xSamples.length
+
+    const rSquared = regression.rSquared
+    const xIntercept = regression.intercept && (-1 * (regression.intercept / regression.slope))
+    return (
+      <div className='regression'>
+        {size === 'SMALL' && _.isFinite(rSquared) &&
+          <div>
+            <span className='label'> r<sup>2</sup></span>
+            <span className={`value ${importance(rSquared)}`}> {rSquared.toFixed(2)}</span>
+          </div>
+        }
+        { size !== 'SMALL' &&
+          <div>
+            <div>
+              <span className='label'> r<sup>2</sup></span>
+              <span className='value'> {regression.rSquared.toFixed(2)}</span>
+            </div>
+            <div>
+              <span className='label'> slope</span>
+              <span className='value'> {regression.slope.toFixed(2)}</span>
+            </div>
+            <div>
+              <span className='label'> x intercept</span>
+              <span className='value'> {xIntercept.toFixed(2)}</span>
+            </div>
+            <div>
+              <span className='label'> y intercept</span>
+              <span className='value'> {regression.intercept.toFixed(2)}</span>
+            </div>
+            <div>
+              <span className='label'> sample count</span>
+              <span className='value'> {sampleSize}</span>
+            </div>
+          </div>
+        }
+      </div>
+    )
+  }
+}
+
+export default class SensitivitySection extends Component {
```

Comment:
Can we not use export defaults? I find it makes it a fair bit more difficult to trace code paths if you use export defaults